framework/src/Volo.Abp.Swashbuckle/Volo/Abp/Swashbuckle/AbpSwashbuckleDocumentFilter.cs

```diff
@@ -8,7 +8,7 @@ namespace Volo.Abp.Swashbuckle;
 
 public class AbpSwashbuckleDocumentFilter : IDocumentFilter
 {
-    protected string[] ActionUrlPrefixes = new[] {"Volo.Abp"};
+    protected string[] ActionUrlPrefixes = new[] {"Volo."};
```

Comment:
But this also hides some APIs like file management (of abp commercial), which are not infrastructural APIs, but application functionalities. right?

Reply:
Yes, that's right. It hides all endpoints whose namespace contains the `Volo.` keyword.